src/aiida_pseudo/cli/install.py

```diff
@@ -148,118 +149,252 @@ def download_sssp(
     return patch_version
 
 
-def download_pseudo_dojo(
-    configuration: 'SsspConfiguration',
+def install_sssp(
     filepath_archive: pathlib.Path,
     filepath_metadata: pathlib.Path,
+    label: str,
+    description: str = '',
     traceback: bool = False
 ) -> None:
-    """Download the pseudopotential archive and metadata for a PseudoDojo configuration to a path on disk.
+    """Install the ``SsspFamily`` and set the recommended cutoffs.
 
-    :param configuration: the PseudoDojo configuration to download.
-    :param filepath_archive: absolute filepath to write the pseudopotential archive to.
-    :param filepath_metadata: absolute filepath to write the metadata archive to.
+    :param filepath_archive: path to the pseudopotential archive.
+    :param filepath_metadata: path to the metadata file.
+    :param label: label of the pseudopotential family group.
+    :param description: description of the pseudopotential family group.
     :param traceback: boolean, if true, print the traceback when an exception occurs.
     """
-    from aiida_pseudo.groups.family import PseudoDojoFamily
+    # pylint: disable=too-many-locals
```

Comment:
does this function really have too many locals?

Reply:
Huh, I could've sworn `pylint` was complaining! Must've been late-night delirium.